client/src/featureform/get.py

```diff
@@ -78,134 +61,105 @@ def get_feature_variant_info(stub, name, variant):
     try:
         for x in stub.GetLabelVariants(iter([searchNameVariant])):
-            format_rows([("NAME: ", x.name),
-            ("VARIANT: ", x.variant), 
-            ("TYPE:", x.type), 
-            ("ENTITY:", x.entity), 
-            ("OWNER:", x.owner), 
-            ("DESCRIPTION:", x.description),
-            ("PROVIDER:", x.provider),
-            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
-            format_pg("SOURCE: ")
-            format_rows([("NAME", "VARIANT"), (x.source.name, x.source.variant)])
-            format_pg("TRAINING SETS:")
-            format_rows("NAME", "VARIANT")
-            for t in x.trainingsets:
-                format_rows(t.name, t.variant)
-            format_pg()
-            return x
+            return Label(
+                name=x.name,
+                source=(x.source.name,x.source.variant),
+                value_type=x.type,
+                owner=x.owner,
+                entity=x.entity,
+                provider=x.provider,
+                description=x.description,
+                location=ResourceColumnMapping("","",""),
+                variant=x.variant,
+                status=ResourceStatus.from_proto(x.status.status),
+                trainingsets=[(f.name,f.variant) for f in x.trainingsets],
+            )
     except grpc._channel._MultiThreadedRendezvous:
         print("Label variant not found.")
 
 def get_source_variant_info(stub, name, variant):
     searchNameVariant = metadata_pb2.NameVariant(name=name, variant=variant)
     try:
         for x in stub.GetSourceVariants(iter([searchNameVariant])):
-            format_rows([("NAME: ", x.name),
-            ("VARIANT: ", x.variant), 
-            ("OWNER:", x.owner),
-            ("DESCRIPTION:", x.description),
-            ("PROVIDER:", x.provider),
-            ("TABLE:", x.table),
-            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
-            format_pg("DEFINITION:")
-            print("TRANSFORMATION")
-            print(x.transformation.SQLTransformation.query)
-            format_pg("SOURCES")
-            format_rows("NAME", "VARIANT")
-            for s in x.transformation.SQLTransformation.source:
-                format_rows(s.name, s.variant)
-            format_pg("PRIMARY DATA")
-            print(x.primaryData.table.name)
-            print("FEATURES:")
-            format_rows("NAME", "VARIANT")
-            for t in x.features:
-                format_rows(t.name, t.variant)
-            format_pg("LABELS:")
-            format_rows("NAME", "VARIANT")
-            for t in x.labels:
-                format_rows(t.name, t.variant)
-            format_pg("TRAINING SETS:")
-            format_rows("NAME", "VARIANT")
-            for t in x.trainingsets:
-                format_rows(t.name, t.variant)
-            format_pg()
-            return x
+            is_transformation = None
+            if x.primaryData.table.name:
```

Comment:
Should add this to its own function that checks  the transformation type instead of having all the `ifs` out here